default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt

```diff
@@ -36,6 +36,33 @@ Part
                     empire = Target.Owner
             ]
 
+        EffectsGroup    // players can order chaos wave used on enemies outposts
+            scope = And [
+                Planet
+                InSystem id = Source.SystemID
+                OrderedBombardedBy condition = Source
+                Or [
+                    OwnedBy affiliation = EnemyOf empire = Source.Owner
+                    Unowned
```

Comment:
if it's already unowned, the message about losing ownership is misleading...

Reply:
Changed it.